lib/plug/templates/inbox_preview/index.html.eex

```diff
@@ -0,0 +1,140 @@
+<html>
```

Comment:
I guess it would be a right thing to place eex in `priv/templates` rather than having them along with Elixir code.